### PR TITLE
Content for My Services including designs for Child Services

### PR DESCRIPTION
**Pull request by beanh66**
Added initial content for the My Services section and included designs for Child Services. Some information is still TBD, pending a meeting later this week. @serenamarie125 Please take a look!

### CODE REVIEW

File: UX/ui-service/MyServices/design.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+* A Table View is the secondary view that should be added once the PatternFly Angular Table is available. The table will show additional fields that may be missing from the List View.
+* Bulk actions can be accessed through the button groups at the top of the list.
+
+### List View
+![Image of List View.](img/Services-ListView.png)
+* The Service Names are links and will bring the user to a service details page.
+* The exact List View fields to be included are TBD.
+* **Implementation Details:** Reverence the Angular PatternFly Expandable List View Component.
+
+### Detail View
```
Comment (beanh66) [inline UX/ui-service/MyServices/design.md]: @serenamarie125 Yes, I changed the format to show there is a Details page which includes a summary View (shown) and in the future a dashboard and topology. Also changed the order to come after expanded list. Let me know what you think.

**Comment by serenamarie125** [comment]
Thanks @beanh66 !

File: UX/ui-service/MyServices/design.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+## View My Services
+* A List View is the default view for My Services and it will not show all fields in this view.
+* A Table View is the secondary view that should be added once the PatternFly Angular Table is available. The table will show additional fields that may be missing from the List View.
+* Bulk actions can be accessed through the button groups at the top of the list.
+
+### List View
+![Image of List View.](img/Services-ListView.png)
+* The Service Names are links and will bring the user to a service details page.
+* The exact List View fields to be included are TBD.
+* **Implementation Details:** Reverence the Angular PatternFly Expandable List View Component.
```
Comment (cshinn) [inline UX/ui-service/MyServices/design.md]: "reverence" should be Reference, but it doesn't matter that much

**Comment by cshinn** [comment]
@beanh66 This all makes sense to me!

File: UX/ui-service/MyServices/design.md (hunk 2 above)
Comment (serenamarie125) [inline UX/ui-service/MyServices/design.md]: @beanh66 can you fix this typo?

**Pull request merged**